src/client/components/RegisterTeaser/RegisterTeaser.component.js

```diff
@@ -12,8 +11,11 @@ export default function RegisterTeaser({ signInURL }) {
       <div className="register-teaser-text">
         <p>
           Do you want to save your results?
-          <br /> <a href={signInURL}>Sign in</a> and check your results any
-          time.
+          <br />{' '}
+          <span onClick={onClick} className="sign-in">
```

Comment:
Why are you passing `onClick` and not e.g. using `<Link>` to navigate to the page instead?

Reply:
we are using history if you think its better to you Link , we will change it

Reply:
we have fixed it